src/Microsoft.DotNet.Web.ProjectTemplates/EmptyWeb-FSharp.fsproj.in

```diff
@@ -10,8 +10,12 @@
     <Compile Include="Program.fs" />
   </ItemGroup>
 
-  <ItemGroup>
+  <ItemGroup Condition="'$(TargetFrameworkOverride)' == ''">
```

Comment:
Was this template previously broken in some scenarios? Like on .NET Framework 4.x?

Reply:
It was fixed [here](https://github.com/aspnet/templating/pull/191/files#diff-470b05a8dc034a8038d7e404d16ae6b5R15) in dev but was broken before